### ISSUE TITLE
Implement training and testing loop

### ISSUE THREAD

**Issue by JankowskiDaniel** [JankowskiDaniel/Neural-Deep-Retina#7]
Implement the main training loop according to the template:

```python
def train_model():
      for e in range(epochs):
            train_epoch()
```

Moreover, implement the script for testing the model.

Both scripts should save intermediate and final results into the specified directory.

**Comment by kapiblue** [comment]
Testing script implemented 47abf8e42f0df81c532edc976909e09d6a34ff8d

**Comment by kapiblue** [comment]
Calculating metrics using torchmetrics and saving to csv implemented: cd98eb2a515eeb9a7a4f2af3414128a92355ece6

**Comment by kapiblue** [comment]
For now, both scripts are done.

We need to be careful with shuffling the datasets and dataloaders.
